changelog.d/20241018_082221_kurtmckee_eliminate_none_return_values.rst

```diff
@@ -0,0 +1,15 @@
+Changes
+-------
+
+-   ``AuthState.introspect_token()`` will no longer return ``None`` if the token is not active.
+
+    Instead, a new exception, ``InactiveTokenError``, will be raised.
+    ``InactiveTokenError`` is a subclass of ``ValueError``.
+
+    Existing code that calls ``AuthState.introspect_token()`` no longer returns ``None``, either,
+    but will instead raise ``ValueError``, a subclass of ``ValueError``,
```

Comment:
`InactiveTokenError` and `InvalidTokenScopesError` are subclasses of `ValueError`...?

Reply:
Right, but the text says that `ValueError` is a subclass of `ValueError`, unless I'm very confused. (And if I am, I'll count that as a request for a rephrase!)

Reply:
Oh! I see now. It's a list of options, of length 3,
- `ValueError`
- a subclass of `ValueError`
- `globus_sdk.GlobusAPIError`

I thought it was a list of length 2,
- `ValueError`, a subclass of `ValueError`
- `globus_sdk.GlobusAPIError`

Can we scratch "a subclass of `ValueError`" from the list, but now on different grounds, which is that it's unnecessary to specify?

Reply:
One last alternate:

> `ValueError` (or a subclass) or `globus_sdk.GlobusAPIError`

?

Reply:
My approval should speak for itself.

But since it doesn't seem to speak up, "yes!" and "w00t!" and other exclamations of joy. I now understand the text and it says everything you wanted it to say!